memcache/memcache.go

```diff
@@ -319,6 +346,33 @@ func (c *Client) Touch(key string, seconds int32) (err error) {
 	})
 }
 
+// Stats returns the stats from all servers.
+func (c *Client) Stats() (map[net.Addr]map[string]string, error) {
```

Comment:
We want a better stats type with field names and actual integers, rather than map[string]string.